### ISSUE TITLE
[Lighthouse / ARIA Allowed Attributes / Helpful]

### ISSUE THREAD

**Issue by armelpingault** [google/WebFundamentals#5936]
**Page Affected:** https://developers.google.com/web/tools/lighthouse/audits/aria-allowed-attributes

**What needs to be done?**

The link on the text "Definition of Roles" https://www.w3.org/TR/wai-aria/roles#role_definitions is broken and is returning an error 404.

- [ ] Fix the link on "Definition of Roles".

**Comment by injms** [comment]
Would https://www.w3.org/WAI/PF/aria/roles#role_definitions_header be the best place to point the currently-404 link to?

**Comment by armelpingault** [comment]
Yes, I think https://www.w3.org/WAI/PF/aria/roles#role_definitions would be the good one ;) Thank you.
